DancingGoat/Scripts/PreviewMode/PreviewMode.js

```diff
@@ -0,0 +1,107 @@
+﻿const previewModeToggleClassname = 'preview-mode-switch';
+const previewModeCookieName = 'IsPreviewMode';
+const previewApiKeyCookieName = 'PreviewApiKey';
+const enterApiKeyFirstMessage = 'Enter your Preview API key';
+const enterApiKeyNextMessage = 'Enter your Delivery Preview API key -- you can find it in Kentico Cloud under Project settings -> API keys';
+const enterApiKeyPromptTitle = 'Preview API key';
+
+const previewModeToggles = document.getElementsByClassName(previewModeToggleClassname);
+const isPreviewModeEnabled = getCookie(previewModeCookieName);
+for (toggle of previewModeToggles) {
+    toggle.checked = isPreviewModeEnabled === 'true' ? true : false;
+}
+
+function togglePreviewMode(isChecked, projectId, isFirstEnterOfApiKey) {
```

Comment:
maybe isFirstEnterOfApiKey rename to isFirstTry ?

Reply:
Fixed